curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/660aef1d5bd6c092f1931830.md

```diff
@@ -0,0 +1,662 @@
+---
+id: 660aef1d5bd6c092f1931830
+title: Step 14
+challengeType: 0
+dashedName: step-14
+---
+
+# --description--
+
+Just regular functions, arrow functions can return values. 
```

Comment:
```suggestion
Just like regular functions, arrow functions can return values. 
```

Reply:
ohh.. hahaha 😆
good catch

english is hard 🤣

Reply:
Harder than JavaScript, tbh.